routes/donations.js

```diff
@@ -0,0 +1,168 @@
+const express = require('express');
+const { keysToCamel } = require('../common/utils');
+const { pool, db } = require('../server/db');
+
+const router = express.Router();
+
+// get all donation rows
+router.get('/', async (req, res) => {
+  try {
+    const allDonations = await pool.query(`SELECT * FROM donations`);
+    res.status(200).json(keysToCamel(allDonations.rows));
+  } catch (err) {
+    res.status(400).send(err.message);
+  }
+});
+
+// get specific donation
+router.get('/:donationId', async (req, res) => {
+  try {
+    const { donationId } = req.params;
+    const donation = await pool.query(`SELECT * from donations WHERE id = $1`, [donationId]);
```

Comment:
Let's use `db` here. There are a couple of other routes you have already that are using the syntax to refer to or also this [example](https://github.com/ctc-uci/find-your-anchor-backend/blob/dev/services/anchorBoxService.js#L24).